frontend/src/views/Game/Ingame.svelte

```diff
@@ -9,10 +9,76 @@
   export let socket: SocketClient
   export let gameData: GameSocketResponse
   export let currentUser: UserResponse
+  enum GameState {
+    PLAYING = 'playing',
+    FLIPPING = 'flipping',
+    VOTING = 'voting',
+    DISPLAY_WINNER = 'winner',
+  }
 
-  const playCard = (card: CardResponse) => {
-    if(!socket.currentUser.cardWizz) {
-      socket.playCard(card)
+  const getWinningPlayer = (game: GameSocketResponse): UserResponse => {
+    const winningPlayer = getPlayers(game).find(user => user.cards.some(c => c.state === CardState.WINNER))
+    if(!winningPlayer) {
+      throw new Error('Could not find winning player')
+    }
+    return winningPlayer
+  }
+
+  const getPlayers = (game: GameSocketResponse): UserResponse[] => {
```

Comment:
maybe this one should live as a method on the socket?

if we have a `currentUser` we just as well could add a `playingPlayers` getter.

Reply:
this will instead be refactored and solved in #70